[2022-02-15 02:28:53,382] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.model.my_new_project.int_orders_pivoted manual__2022-02-15T02:27:57.149490+00:00 [queued]>
[2022-02-15 02:28:53,449] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.model.my_new_project.int_orders_pivoted manual__2022-02-15T02:27:57.149490+00:00 [queued]>
[2022-02-15 02:28:53,449] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 02:28:53,450] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-15 02:28:53,451] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 02:28:53,526] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): model.my_new_project.int_orders_pivoted> on 2022-02-15 02:27:57.149490+00:00
[2022-02-15 02:28:53,533] {standard_task_runner.py:52} INFO - Started process 801 to run task
[2022-02-15 02:28:53,552] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dbt_training_bytecode', 'model.my_new_project.int_orders_pivoted', 'manual__2022-02-15T02:27:57.149490+00:00', '--job-id', '73', '--raw', '--subdir', 'DAGS_FOLDER/dbt-***.py', '--cfg-path', '/tmp/tmpr2mpluu_', '--error-file', '/tmp/tmp4vci4zpk']
[2022-02-15 02:28:53,553] {standard_task_runner.py:77} INFO - Job 73: Subtask model.my_new_project.int_orders_pivoted
[2022-02-15 02:28:53,724] {logging_mixin.py:109} INFO - Running <TaskInstance: dbt_training_bytecode.model.my_new_project.int_orders_pivoted manual__2022-02-15T02:27:57.149490+00:00 [running]> on host de7a240d0d46
[2022-02-15 02:28:53,912] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Bytecode
AIRFLOW_CTX_DAG_ID=dbt_training_bytecode
AIRFLOW_CTX_TASK_ID=model.my_new_project.int_orders_pivoted
AIRFLOW_CTX_EXECUTION_DATE=2022-02-15T02:27:57.149490+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-15T02:27:57.149490+00:00
[2022-02-15 02:28:53,922] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-15 02:28:53,929] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n            dbt --no-write-json run --target dev --models int_orders_pivoted             --profiles-dir /opt/***/dbt/dbt-training-bytecode-1 --project-dir /opt/***/dbt/dbt-training-bytecode-1\n            ']
[2022-02-15 02:28:53,944] {subprocess.py:85} INFO - Output:
[2022-02-15 02:29:05,958] {subprocess.py:89} INFO - 02:29:05  Running with dbt=1.0.1
[2022-02-15 02:29:06,319] {subprocess.py:89} INFO - 02:29:06  [WARNING]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2022-02-15 02:29:06,320] {subprocess.py:89} INFO - There are 2 unused configuration paths:
[2022-02-15 02:29:06,321] {subprocess.py:89} INFO - - models.jaffle_shop.marts.core
[2022-02-15 02:29:06,322] {subprocess.py:89} INFO - - models.jaffle_shop.staging
[2022-02-15 02:29:06,322] {subprocess.py:89} INFO - 
[2022-02-15 02:29:06,348] {subprocess.py:89} INFO - 02:29:06  Found 7 models, 11 tests, 0 snapshots, 0 analyses, 367 macros, 0 operations, 1 seed file, 3 sources, 0 exposures, 0 metrics
[2022-02-15 02:29:06,355] {subprocess.py:89} INFO - 02:29:06
[2022-02-15 02:29:10,362] {subprocess.py:89} INFO - 02:29:10  Concurrency: 4 threads (target='dev')
[2022-02-15 02:29:10,364] {subprocess.py:89} INFO - 02:29:10
[2022-02-15 02:29:10,517] {subprocess.py:89} INFO - 02:29:10  1 of 1 START view model dbt_bytecode_training.int_orders_pivoted................ [RUN]
[2022-02-15 02:29:13,391] {subprocess.py:89} INFO - 02:29:13  1 of 1 OK created view model dbt_bytecode_training.int_orders_pivoted........... [SUCCESS 1 in 2.87s]
[2022-02-15 02:29:13,485] {subprocess.py:89} INFO - 02:29:13
[2022-02-15 02:29:13,487] {subprocess.py:89} INFO - 02:29:13  Finished running 1 view model in 7.13s.
[2022-02-15 02:29:13,494] {subprocess.py:89} INFO - 02:29:13
[2022-02-15 02:29:13,496] {subprocess.py:89} INFO - 02:29:13  Completed successfully
[2022-02-15 02:29:13,499] {subprocess.py:89} INFO - 02:29:13
[2022-02-15 02:29:13,503] {subprocess.py:89} INFO - 02:29:13  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2022-02-15 02:29:14,254] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-15 02:29:14,286] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=dbt_training_bytecode, task_id=model.my_new_project.int_orders_pivoted, execution_date=20220215T022757, start_date=20220215T022853, end_date=20220215T022914
[2022-02-15 02:29:14,325] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-15 02:29:14,364] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
